src/test/java/org/fogbowcloud/saps/engine/core/archiver/storage/SwiftPermanentStorageTest.java

```diff
@@ -269,80 +269,78 @@ public void failureTestWhenTryingToArchiveFailureTaskWithArchiverInDebugMode() t
         Mockito.verify(swiftAPIClient, Mockito.times(1)).deleteFile(MOCK_CONTAINER_NAME, MOCK_SWIFT_FOLDER_PREFIX + "/" + Files.Task01.PROCESSING_FILE);
     }
 
     @Test
-    public void testToDeleteFailureTaskWithoutArchiverInDebugMode()
-        throws IOException, PermanentStorageException {
+    public void testToDeleteFailureTaskWithoutArchiverInDebugMode() throws Exception {
         SwiftAPIClient swiftAPIClient = mock(SwiftAPIClient.class);
         Properties properties = createDefaultPropertiesWithoutArchiverInDebugMode();
         PermanentStorage permanentStorage = new SwiftPermanentStorage(properties, swiftAPIClient);
 
-        Mockito.when(swiftAPIClient.listFilesWithPrefix(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.eq(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task02))).thenReturn(filesTask02);
+        Mockito.when(swiftAPIClient.listFiles(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.eq(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task02))).thenReturn(filesTask02);
         Mockito.doAnswer((i) -> {
             return null;
         }).when(swiftAPIClient).deleteFile(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.argThat(new StartsWith(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task02)));
 
-        boolean deleteTask02 = permanentStorage.delete(task02);
+        boolean deleteTask02 = permanentStorage.archive(task02);
 
         Assert.assertEquals(deleteTask02, true);
-        Mockito.verify(swiftAPIClient, Mockito.times(1)).listFilesWithPrefix(MOCK_CONTAINER_NAME, MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task02);
+        Mockito.verify(swiftAPIClient, Mockito.times(1)).listFiles(MOCK_CONTAINER_NAME, MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task02);
         Mockito.verify(swiftAPIClient, Mockito.times(1)).deleteFile(MOCK_CONTAINER_NAME, MOCK_SWIFT_FOLDER_PREFIX_DEBUG_FAILED_TASKS + "/" + Files.Task02.INPUTDOWNLOADING_FILE);
     }
 
     @Test
     public void testToDeleteSuccessfulTaskButListFilesWithPrefixMethodReturnsEmptyListWithoutArchiverInDebugMode()
-        throws IOException, PermanentStorageException {
+        throws Exception {
         SwiftAPIClient swiftAPIClient = mock(SwiftAPIClient.class);
         Properties properties = createDefaultPropertiesWithoutArchiverInDebugMode();
         PermanentStorage permanentStorage = new SwiftPermanentStorage(properties, swiftAPIClient);
 
-        Mockito.when(swiftAPIClient.listFilesWithPrefix(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.eq(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task01))).thenReturn(new ArrayList<String>());
+        Mockito.when(swiftAPIClient.listFiles(Mockito.eq(MOCK_CONTAINER_NAME), Mockito.eq(MOCK_SWIFT_FOLDER_PREFIX + "/" + Dirs.Task01))).thenReturn(new ArrayList<String>());
         Mockito.doAnswer((i) -> {
             return null;
         }).when(swiftAPIClient).deleteFile(Mockito.anyString(), Mockito.anyString());
 
-        boolean deleteTask01 = permanentStorage.delete(task01);
+        boolean deleteTask01 = permanentStorage.archive(task01);
```

Comment:
Change to `permanentStorage.delete(task01)`.

Reply:
Done